src/Samples/NetCoreServer/NetCoreServer.csproj

```diff
@@ -7,6 +7,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore" Version="2.1.7" />
+    <PackageReference Include="System.ServiceModel.Primitives" Version="4.8.1" />
+    <PackageReference Include="System.ServiceModel.Http" Version="4.8.1" />
```

Comment:
4.9.0 version of the packages are now available.

Reply:
Updated to 4.9.0